augly/image/functional.py

```diff
@@ -1759,6 +1759,56 @@ def shuffle_pixels(
     return imutils.ret_and_save_image(aug_image, output_path)
 
 
+def skew(
+    image: Union[str, Image.Image],
+    output_path: Optional[str] = None,
+    level: float = 0.5,
+    axis: str = "x-axis",
+    metadata: Optional[List[Dict[str, Any]]] = None,
+) -> Image.Image:
+
+    """
+    Skew image with respect to X-axis or Y-axis
+
+    @param image: the path to an image or a variable of type PIL.Image.Image
+        to be augmented
+
+    @param output_path: the path in which the resulting image will be stored.
+        If None, the resulting PIL Image will still be returned
+
+    @param level: the level of skew to apply to the image; a larger value means the skew
+        will be more intense
```

Comment:
Could you please add to this docstring what are the recommended ranges for this parameter? Since the default value of `level` is 0.5, does that mean that the range is from 0-1, or can I input successfully a value of 100?

Reply:
Yes suitable range is between 0-1 for level !!